src/main/java/edu/unc/lib/boxc/migration/cdm/services/ListProjectsService.java

```diff
@@ -77,8 +79,14 @@ public JsonNode listProjects(Path directory) throws Exception {
                 }
             }
         }
-        log.debug(arrayNode.toString());
 
+        // count number of archived projects
+        int numberArchivedProjects = countArchivedProjects(directory);
+        ObjectNode objectNode = mapper.createObjectNode();
+        objectNode.put(ARCHIVED_PROJECTS, numberArchivedProjects);
+        arrayNode.add(objectNode);
```

Comment:
Hm, i think it's going to be confusing if the result from this service is a list of projects, and then one of the entries in the list is just a count archived projects. Will probably also mess up the table in the admin UI. Maybe we can add this count to one of the status commands instead?

I'm also wondering if this service should know how to list archived projects in the .archived subdirectory, and include them in the regular results. Otherwise it seems like we'll never have any results with status = archived. It'd probably just mean making lines 60-81 into a helper method and then calling it on both the base directory and the archived subdirectory. If we do that, we might want to add an option so that archived projects only get included if the option is set, like `--include-archived`.

Reply:
yes you're right, it wouldn't make sense in the status commands, so removing the stat makes sense